### DISCUSSION TITLE
I feel like the way the date is rounded off is wrong

### DISCUSSION THREAD

**Discussion by lazandrei19** [community/community#6611] [Other Feature Feedback, Questions, & Ideas]
For example, I was browsing https://github.com/github/covid-19-repo-data and it says that files like CONTRIBUTING and CODE_OF_CONDUCT where last updated 2years ago, but it has been only 1.5 years. I feel like saying "1 year ago" would be better
(also it wouldn't make me question my life and wonder if it's truly been two years since the start of lockdown lol)

**Comment by Hubcapp** [comment]
I agree, half a year is a lot to round up, since it is about 2/3rds a percent of an expected human lifespan. Rounding up like Github currently does, 1%.

It caused me to believe that an anniversary of our project had already occurred when it is actually 5 months away.

**Comment by shivvamm** [comment]
I understand your perspective. Sometimes, more rounded time descriptions can be more relatable and easier to understand. Saying "1 year ago" instead of "2 years ago" would indeed provide a clearer and more accurate sense of time, especially when it comes to recent events like the COVID-19 lockdown. It's all about presenting information in a way that resonates better with users and avoids unnecessary confusion or surprise.
